docs/conf.py

```diff
@@ -31,6 +31,8 @@ def real_dir_name(p, n=1):
     print p
     return p
 
+if not hasattr(sys, 'real_prefix'):
+    raise RuntimeError('A virtualenv must be active')
```

Comment:
This change would be in a separate PR against the documentation branch and it should use the same commit message as the original commit. When I squash down the doc branch, I'll merge those two commits.